docs/source/cli_help/squash.rst

```diff
@@ -0,0 +1,23 @@
+.. role:: bash(code)
+
+.. _squash:
+
+squash
+------
+Usage:
+
+.. code-block:: shell
+
+    git machete squash [-f|--fork-point=<fork-point-commit>]
+
+Squashes the commits belonging uniquely to the current branch into a single commit.
+The chunk of the history to be squashed starts at the automatically computed fork point of the current branch by default, but can also be set explicitly by :bash:`--fork-point`.
+See :bash:`git machete help fork-point` for more details on meaning of the "fork point".
```

Comment:
Link + cursive instead of `"`

Reply:
I will skip making text italic, because  link itself requires `` that are used of italics so I can do either one, but not both. There might be some workaround, but it's not worth it at the moment IMO.